README.md

```diff
@@ -41,9 +39,17 @@ You can tell me what to say with:
   localhost:8080/gob?text=WHAT_TO_SAY&limit=NUMBER
 ```
 
-_websvc_ may call both _wordsvc_ and _gensvc_ to satisfy a request.  
+_web_ may call both _word_ and _gen_ to satisfy a request.
 
-_wordsvc_ and _gensvc_ implement RPC-ish interfaces with HTTP and JSON.
+_word_ and _gen_ implement RPC-ish interfaces with HTTP and JSON.
 
 All three services are implemented in Go with no shared code.  They
 may be built and run independently.
+
+If you want to run these programs locally, you'll need to
+[Go 1.6 or later](https://golang.org/dl).  However, this isn't
+necessary to deploy them, since docker can take care of those
+dependencies.  Docker, however, is not even required to deploy these
+apps, since we have already published images to
+[https://hub.docker.com/u/gobsvc/].
```

Comment:
I think it's worth making a distinction here that docker isn't required _locally_ to deploy these apps.  We still use docker to deploy the code to prod -- you just don't need to be able to build the images locally because they're already published.